### DISCUSSION TITLE
OsmandMapCreator download

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#11194] [Q&A] [answered]
I am not able to download OsmandMapCreator. Each time I get an error message "...because the source file could not be read. Try again later, or contact the server administrator." See attached screen shot.

The link I have is http://download.osmand.net/latest-night-build/OsmAndMapCreator-main.zip

I have tried again later for several days, but can't see any way to contact anyone on that part of GitHub.

Can anyone on this group help or point me in the right direction?

Thanks

![OsmandMapCreator Download Error](https://user-images.githubusercontent.com/69813072/111692029-dfcfbb80-8826-11eb-8bf3-f27a8493a3bf.png)

**Comment by vshcherb** [comment, marked as answer]
Utilities like "wget http://download.osmand.net/latest-night-build/OsmAndMapCreator-main.zip" works fine, probably you need to use https

**Reply by ghost** [reply]
Thanks vshcherb, that works a treat
